donate/utility/acoustic/acoustic.py

```diff
@@ -107,10 +108,23 @@ def __init__(self, client_id, client_secret, refresh_token, server_number):
         super().__init__(client_id, client_secret, refresh_token, server_number)
 
     def _call_xt(self, xml):
+        send_retries = 3
         logger.debug("Request: %s" % xml)
-        response = self.session.post(
-            self.api_xt_endpoint, data=force_bytes(xml), timeout=10,
-        )
+        for n in range(send_retries):
```

Comment:
Similar to the _call function above, we are now trying the post request that is sending a message, if there is a connection error which is the error that Dan G mentioned he experienced, we are then logging that we could not send the message due to a connection error.